src/h5web/vis-packs/core/shared/AxisSystem.tsx

```diff
@@ -45,6 +45,7 @@ function AxisSystem(props: Props) {
     <>
       <Html
         className={styles.axisSystem}
+        fullscreen
```

Comment:
I added a `fullscreen` prop in accordance with drei's version.

When using `Html` to annotate things, we don't want to the annotating `div` to take the canvas dimensions.

Reply:
I'm wondering whether the fullscreen behaviour should depend on `followCamera`. It seems the two props are mutually exclusive.

Reply:
> It seems the two props are mutually exclusive.

For our case yesy. But take the Braggy radial grid for example: you want it to take the whole heatmap while still following the camera.

Reply:
On second thought, the prop is perhaps not really adapted in our case. I will remove it and let the caller set the width/height.